### PR TITLE
Props corrige2

### PR DESCRIPTION
**Pull request by harlouci**
Bonne branche props

### CODE REVIEW

**Comment by nouiz** [comment]
this PR need a rebase.

**Comment by nouiz** [comment]
we can merge when travis pass.

File: theano/tensor/extra_ops.py [hunk 1]
```diff
@@ -12,16 +12,11 @@
 
 class CumsumOp(theano.Op):
     # See function cumsum for docstring
+    __props__ = ("axis")
```
Comment (nouiz) [inline theano/tensor/extra_ops.py]: this don't make a tuple, this create just a string. use `("axis",)`, the comma is important. That is what make travis fail.

Comment (nouiz) [inline theano/tensor/extra_ops.py]: idem elsewhere in this file.

**Comment by nouiz** [comment]
travis fail

**Comment by harlouci** [comment]
Travis fails, but in my local depository "flake8 --ignore E501
theano/tensor/extra_ops.py" signals no error.

2015-07-17 9:21 GMT-04:00 Frédéric Bastien <notifications@github.com>:

**Comment by lamblin** [comment]
The master was failing at the time where this PR got merged. Since Travis is overwhelmed at the moment, I would not try to trigger a new merge at the moment, so I would be ready to merge it.
@nouiz is that fine for you?

**Pull request merged**